src/rt-timeseries/RtTimeseriesContainer.tsx

```diff
@@ -0,0 +1,86 @@
+import { ascending, isoParse } from "d3";
+import mapValues from "lodash/mapValues";
+
+import RtTimeseries, { Record } from "./RtTimeseries";
+
+const convertToDate = isoParse;
+
+function dateExists(value: { date: Date | null }): value is Record {
+  // date formatting function may return null with invalid input
+  // so we do need a type guard to filter the results
+  return value.date != null;
+}
+
+const RtTimeseriesContainer: React.FC = () => {
+  // TODO: this is dummy data
```

Comment:
Thanks for the thoughtful consideration of this in the PR description. I am fine placing this into a `__tests__` directory as a place where we could manually run the test, but I am becoming less comfortable with checking this kind of thing into master now that, as you said, release cadence is increasing, and user counts are climbing.

Reply:
what if I just commented it out for now, with a louder TODO? putting random stuff in a `__tests__` directory will start causing problems with Jest once we get that up and running

Reply:
Sure seems reasonable for now.